WalletWasabi.Fluent/Views/AddWallet/WelcomePageView.axaml

```diff
@@ -228,23 +228,23 @@
           <TextBlock Grid.ColumnSpan="2" Grid.Column="0" Grid.Row="0" Text="Coinjoin Fees" Classes="title1" HorizontalAlignment="Center" VerticalAlignment="Center" />
 
           <StackPanel Grid.Column="0" Grid.Row="1">
-            <TextBlock Name="CoinjoinFeeTextBlock" Text="Service Fee" Classes="title2" />
-            <TextBlock Text="You will pay a flat fee of 0.3% for every coin you receive." Classes="text" />
+            <TextBlock Name="CoinjoinFeeTextBlock" Text="Coordination Fee" Classes="title2" />
+            <TextBlock Text="You will pay a fee of 0.3% only for new coins you coinjoin." Classes="text" />
           </StackPanel>
 
           <StackPanel Grid.Column="1" Grid.Row="1">
-            <TextBlock Name="MiningFeeTextBlock" Text="Network Fee" Classes="title2" />
+            <TextBlock Name="MiningFeeTextBlock" Text="Transaction Fee" Classes="title2" />
             <TextBlock Text="The market standard in mining fee always applies." Classes="text" />
```

Comment:
I guess this one also should be improved.

```suggestion
            <TextBlock Text="You will pay also a mining fee always. (for every coinjoin)" Classes="text" />
```

Reply:
"Mining fees are always required for each coinjoin transaction"
or
"Mining fees are always paid for each coinjoin transaction"

Reply:
@turbolay's suggestion is consistant with the previous text block.
@Kruw's second suggestion sounds also better than the current and @yahiheb's suggestion.

Reply:
> You will also pay mining fees for every coinjoin.

ACK.

Reply:
done